NOTICE

```diff
@@ -1,6 +1,6 @@
 Druid - a distributed column store.
-Copyright 2012-2015 Metamarkets Group Inc.
-Copyright 2015 Yahoo! Inc.
+Copyright 2012-2016 Metamarkets Group Inc.
+Copyright 2015-2016 Yahoo! Inc.
```

Comment:
I just updated dates in this one, I think any new (C) notices should be a separate PR